src/components/Input/Input.md

```diff
@@ -1,13 +1,17 @@
 ### Default Style
 ```jsx
-<Input type="text" name="text-demo" label="Text Input" note="This is a note about this field." margin={{ bottom: 'md' }} />
+<Input type="text" name="text-demo" label="Text Input" note="This is a note." margin={{ bottom: 'md' }} />
 <Input type="text" name="text-demo-disabled" label="Text Input (Disabled)" disabled />
 ```
 
 ### Input Inline Validation
 ```jsx
-<Input type="text" name="valid-input" label="First Name" variant="valid" note={<><strong>First Name</strong> is valid</>} margin={{bottom: 'md'}} />
-<Input type="text" name="warning-input" label="Password" variant="warning" note={<>Your <strong>Password</strong> is weak</>} margin={{bottom: 'md'}} />
-<Input type="text" name="error-input" label="Username" variant="invalid" note={<>The <strong>Username</strong> you entered is taken</>} margin={{bottom: 'md'}} />
-<Input type="text" name="info-input" label="Description" variant="info" note={<>The <strong>Description</strong> tells users more about this stuff.</>} margin={{bottom: 'md'}} />
+<Input type="text" name="valid-input" label="First Name" variant="valid" 
```

Comment:
We generally expand the tag to one property per line if the tag does not fit on one line. I'm fine with either approach, but thought I would mention it.

Reply:
Good call -- I've changed them to one per line.